Assets/BU/QuizExample/Scripts/UI/QuizResultUI/StudentUI/StudentPostTestQuizResultPanel.cs

```diff
@@ -42,11 +53,20 @@ private void Start()
             SetQuestionAmountText(QuestionAmount);
             SetPostTestQuizScoreText(CurrentScore, QuestionAmount);
             SetPreTestQuizScoreText(PreTestScore, QuestionAmount);
+            
+            
         }
 
         public override void OnCustomDataReceive(byte[] data)
         {
-            Debug.Log($"NPA-data:{data}");
+            Debug.Log($"NPA-data:{data[0]}");
+            // RRTT Additional
+            bossReference = GameObject.FindGameObjectWithTag("BossReference");
+            bossList = bossReference.GetComponent<BossList>();
+            GameObject boss = Instantiate(bossList.bossPrefabs[data[0]].gameObject, bossPosition);
+            boss.transform.localScale = new Vector3(4.5f, 4.5f, 4.5f);
```

Comment:
อันนี้เป็น suggestion เฉยๆ นะ จะแก้รึเปล่าก็ได้ 
ตรง scale ถ้าเก็บเป็น variable ไว้ จะยืดหยุ่นกว่าถ้ามีที่อยากเปลี่ยนหลายจุด
```suggestion
            float scale = 4.5f;
            boss.transform.localScale = new Vector3(scale, scale, scale);
```